docs/dev_ops/release_procedure.md

```diff
@@ -0,0 +1,20 @@
+# Releasing in Modalitites
+This tutorial describes the procedure to release a new version of the Modalitites package.
+
+## Release Types
+We follow the release types as defined by [Semantic Versioning](https://semver.org/). The version number is defined as `MAJOR.MINOR.PATCH` where:
+- `MAJOR` is incremented when you make incompatible API changes,
+- `MINOR` is incremented when you add functionality in a backwards-compatible manner, and
+- `PATCH` is incremented when you make backwards-compatible bug fixes.
+
+
+## Releasing a new Modalitites version
+1. Update the version number in the pyproject.toml file.
+2. Commit the version bump following the convention `git commit --no-verify -m "<version number>"`.
+   The `--no-verify` flag is used to skip the pre-commit hooks.
+3. Run `git push` to push the changes to the remote repository.
+4. Make sure that all the tests pass in the main branch.
```

Comment:
I think the current step 4 should be done as the very first step, before increasing the version number. Otherwise, if the tests fail at this point, this is a problem because the version bump has already been commited.

I would also demand that not only the tests should pass, but that the whole repository is in a clean, well-functioning state. Maybe by saying something like "Make sure that the main branch is in a clean state. In particular, all tests should pass".